test/fixtures/await-arrow/await-array/actual.js

```diff
@@ -1,4 +1,4 @@
 fn() -/>
   x <- [p1, p2]
   const y = await [p1, p2]
-  <- [for p of ps: p]
+  <- [for now p of ps: p]
```

Comment:
hmm, this is pretty non-idiomatic... 
should either be `for const p` or `for elem p in ps`